Crypter.Test/Integration_Tests/FileTransfer_Tests/DownloadFileTransfer_Tests.cs

```diff
@@ -71,9 +72,9 @@ public async Task OneTimeTearDown()
       [Test]
       public async Task Download_Anonymous_File_Transfer_Works()
       {
-         (EncryptionStream encryptionStream, byte[] keyExchangeProof) = TestData.GetDefaultEncryptionStream();
+         (Func<EncryptionStream> encryptionStreamOpener, byte[] keyExchangeProof) = TestData.GetDefaultEncryptionStream();

```

Comment:
Unless it's helpful for you to read the individual types, the code is much cleaner as:
```suggestion
         var (encryptionStreamOpener, keyExchangeProof) = TestData.GetDefaultEncryptionStream();
```